liquibase-standard/src/main/java/liquibase/change/core/AddForeignKeyConstraintChange.java

```diff
@@ -55,24 +51,6 @@ public class AddForeignKeyConstraintChange extends AbstractChange {
     private String onUpdate;
     private String onDelete;
 
-
-    @Override
-    protected String[] createSupportedDatabasesMetaData(
```

Comment:
This was totally wrong and was causing https://github.com/liquibase/liquibase-docs/issues/105